frontend/src/helpers/GetCookie.js

```diff
@@ -0,0 +1,16 @@
+function getCookie(name) {
+  let cookieValue = null;
+  if (document.cookie && document.cookie !== "") {
+    const cookies = document.cookie.split(";");
+    for (const element of cookies) {
+      const cookie = element.trim();
+      if (cookie.substring(0, name.length + 1) === name + "=") {
+        cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+        break;
+      }
+    }
+  }
+  return cookieValue;
+}
```

Comment:
@muhammad-ali-e Suggestion: We can avoid nested if conditions by modifying the code like below,

```
function getCookie(name) {
    let cookieValue = null;

    if (!document.cookie) {
      return cookieValue;
    }

    const cookies = document.cookie.split(";");
    for (const element of cookies) {
      const cookie = element.trim();
      if (cookie.substring(0, name.length + 1) === name + "=") {
        cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
        break;
      }
    }
    return cookieValue;
  }
```

Reply:
Replaced this logic with package  https://www.npmjs.com/package/js-cookie